modules/monitoring/files/check_reverse_dns.py

```diff
@@ -78,8 +78,14 @@ def check_records(hostname):
         for nameserver in nameserversans:
             nameserver = str(nameserver)
             nameservers.append(nameserver)
-            if nameserver.endswith('.ns.cloudflare.com.') or nameserver.endswith('.dreamhost.com.') or nameserver.endswith('.ns.porkbun.com.'):
-                uses_cf_at_root = True
+            flatten_manadatory_providers = [
+                'ns.cloudlfare.com',
+                'dreamhost.com',
+                'porkbun.com',
```

Comment:
```suggestion
                'ns.porkbun.com',
```
For what it was before anyway.